ubuntudesign/canonical-vanilla-theme
# Add demo page for components

# Done
- Added devDeps to package.json
- Added demo page
- Linked the demo page to the generated css
- Changed _theme to theme.scss to auto generate

## QA
- Run ```npm install```
- Run ```gulp build```
- Run ```google-chrome demo/index.html```
- Check the theme styles have been applied
